README.md

```diff
@@ -21,6 +21,21 @@ registry.example.com/nginx/nginx:latest
 ```
 
 ## NOTICE
+Kubernetes APIs upgrade
+
+>**Deprecated APIs in v1.22** https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-22 Because the end of support for some beta APIs the code received some important updates. Now you need to specify a signer for your Certificate Signing Request. You can take a look on the official documentation about API changes https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-22.
+
+>**K8s prior to v1.19** Will no longer work due to the admissionregistration.k8s.io/v1beta1 api deprecation.
```

Comment:
This needs to be rephrased a bit to point out that ImageSwap v1.5.0+ drops support for k8s versions below v1.19. Also include something like `For deployment on K8s version v1.19 and before, please use ImageSwap v1.4.x`